exporter/src/main/as/flump/export/PreviewController.as

```diff
@@ -111,13 +113,18 @@ public class PreviewController
         const scale :Number = MathUtil.clamp(Number(_atlasPreviewWindow.scale.text), 0.001, 1);
         const border :int = Math.max(0, int(_atlasPreviewWindow.border.text));
 
-        const atlases :Vector.<Atlas> = TexturePacker.withLib(_lib)
-            .baseScale(scale)
-            .borderSize(border)
-            .optimizeForSpeed(exportConf.optimize == ExportConf.OPTIMIZE_SPEED)
-            .quality(exportConf.quality)
-            .createAtlases();
-
+        try {
+            const atlases:Vector.<Atlas> = TexturePacker.withLib(_lib)
+                    .baseScale(scale)
+                    .borderSize(border)
+                    .quality(exportConf.quality)
+                    .maxAtlasSize(exportConf.maxAtlasSize)
+                    .createAtlases();
+        }
+        catch (err : Error) {
+            ErrorWindowMgr.showErrorPopup("Error", err.toString(), _atlasPreviewWindow);
+            return;
+        }
```

Comment:
show error if packing fail, for example is the user specified too little atlas size